### PR TITLE
start to merge in swift-lite stuff

### PR DESCRIPTION
**Pull request by rpedde**

### CODE REVIEW

File: wipe.sh [hunk 1]
```diff
@@ -16,3 +16,13 @@ for d in $(nova list | grep ${job} | awk '{ print $4 }' ); do
     echo "Deleting ${d}"
     nova delete ${d}
 done
+
+for d in $(quantum subnet-list | grep ${job} | awk '{ print $2 }' ); do
+    echo "Deleting quantum subnet ${d}"
```
Comment (breu) [inline wipe.sh:21]: Please protect the "public" subnet here from deletion

File: wipe.sh [hunk 2]
```diff
@@ -16,3 +16,13 @@ for d in $(nova list | grep ${job} | awk '{ print $4 }' ); do
...
+    quantum subnet-delete ${d}
+done
+
+for d in $(quantum net-list | grep ${job} | awk '{ print $2 }' ); do
+    echo "Deleting quantum network ${d}"
+    quantum net-delete ${d}
```
Comment (breu) [inline wipe.sh:27]: Please protect the "public" network here from deletion

File: wipe.sh (hunk 1 above)
Comment (rpedde) [inline wipe.sh:21]: no

File: wipe.sh (hunk 2 above)
Comment (rpedde) [inline wipe.sh:27]: no

File: wipe.sh (hunk 1 above)
Comment (breu) [inline wipe.sh:21]: yes

File: wipe.sh (hunk 2 above)
Comment (breu) [inline wipe.sh:27]: yes

**Pull request closed without merging**